packages/SwingSet/test/util.js

```diff
@@ -45,7 +45,10 @@ export function dumpKT(kernel) {
   }
 }
 
-export function buildDispatch(onDispatchCallback = undefined) {
```

Comment:
optionality conveyed by jsdoc so they type checker doesn't take this as a hint that the value is only ever `undefined`